Formula/ydotool.rb

```diff
@@ -0,0 +1,67 @@
+class Ydotool < Formula
+  desc "Generic command-line automation tool (no X!)"
+  homepage "https://github.com/ReimuNotMoe/ydotool"
+  url "https://github.com/ReimuNotMoe/ydotool/archive/refs/tags/v1.0.4.tar.gz"
+  sha256 "ba075a43aa6ead51940e892ecffa4d0b8b40c241e4e2bc4bd9bd26b61fde23bd"
+  license "AGPL-3.0-only"
+  head "https://github.com/ReimuNotMoe/ydotool.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  bottle do
+    root_url "https://github.com/ublue-os/homebrew-tap/releases/download/ydotool-1.0.4"
+    sha256 cellar: :any_skip_relocation, x86_64_linux: "66a00b7830a0c9703c7ae073fa3f45f2f2409a84aeeae463c77262ecd330e7fa"
+  end
```

Comment:
We don't have CI setup to build this yet, we can remove this and then merge when green